addons/account/views/account_portal_templates.xml

```diff
@@ -122,14 +122,10 @@
 
                     <t t-set="entries">
                         <div class="d-flex flex-column gap-4">
-                            <div class="flex-basis-100 flex-basis-sm-50 flex-basis-lg-100 order-1 order-lg-0">
-                                <div class="o_download_pdf d-flex gap-1 flex-lg-column flex-xl-row">
-                                    <div class="flex-grow-1">
-                                        <a class="btn btn-secondary d-block o_download_btn" t-att-href="invoice.get_portal_url(report_type='pdf', download=True)" title="Download"><i class="fa fa-download"/> Download</a>
-                                    </div>
-                                    <div class="flex-grow-1">
-                                        <a class="btn btn-secondary d-block o_print_btn o_portal_invoice_print" t-att-href="invoice.get_portal_url(report_type='pdf')" id="print_invoice_report" title="Print" target="_blank"><i class="fa fa-print"/> Print</a>
-                                    </div>
+                            <div>
```

Comment:
Question: is this `<div>` necessary?

Reply:
There might be a better way ^^

As the parent has a gap-4, I use this div to be able to xpath INSIDE of it to avoid having that gap-4 when xpathing extra buttons, will look for something else !